SingularityUI/app/components/common/input/FormField.cjsx

```diff
@@ -0,0 +1,30 @@
+Utils = require '../../../utils'
+
+FormField = React.createClass
+
+    defaultSize: 50
+
+    getSize: ->
+        if @props.size
+            return @props.size
+        else
+            return @defaultSize
+
+    getDisabled: ->
+        if @props.disabled
```

Comment:
This function used to do something different, then I changed around some functionality and forgot about it. That is a very good point.